pkg/pgsql/server/bmessages/ready_for_query.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2021 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bmessages
+
+import (
+	"bytes"
+	"encoding/binary"
+)
+
+func ReadyForQuery() []byte {
```

Comment:
Is it really mandatory or is there an advantage of having each of these messages in their own dedicated go files instead of all of them in a single file (as they seem relatively small and serving a similar purpose)?

Reply:
There are messages that are complex. So I prefer to keep them separated to eliminate at the beginning the risk of large files.